docs/4.4/enterprise/ssh-rbac.md

```diff
@@ -290,7 +290,6 @@ As list of all options is below.
       - read
       - update
       - delete
-    # Can view active sessions.
```

Comment:
As far as I experienced, the `session` access is used to control session recordings. Example use case: If session `read` is false, users can't view the recordings